modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/form.js

```diff
@@ -239,19 +239,24 @@ AUI.add(
 				_valueEditingLanguageId: function() {
 					var instance = this;
 
-					var portletNamespace = instance.get('portletNamespace');
+					var languageId;
 
-					var languageId = instance._getURLParameter(
```

Comment:
Apparently, this method isn't used in any other place, thus, if this is the case, then it should be deleted.

Reply:
Indeed!